simplyblock_core/utils/__init__.py

```diff
@@ -1782,7 +1782,7 @@ def get_storage_node_api_log_type(mgmt_ip, name):
         log_config = container.attrs["HostConfig"]["LogConfig"]
         if log_config and log_config["Type"]:
             return log_config["Type"]
-    except docker.errors.NotFound:
+    except (docker.errors.NotFound, docker.errors.DockerException, Exception):
```

Comment:
why silently catch the `Exception`? don't we want to fail if other exceptions are received?